lib/stream.js

```diff
@@ -1,11 +1,20 @@
 'use strict';
 
-module.exports = Stream;
-
 const EE = require('events');
 const util = require('util');
 
+function Stream() {
+  EE.call(this);
+}
 util.inherits(Stream, EE);
```

Comment:
Why can't we put this into the internal/* file so that the code is more exactly like what we're removing, instead of wrapping it?